subsys/net/lib/sntp/sntp_simple.c

```diff
@@ -17,7 +17,7 @@ int sntp_simple(const char *server, uint32_t timeout, struct sntp_time *time)
 	uint64_t deadline;
```

Comment:
There is now this sentence in the commit message, it could be removed before merging as this PR is now self contained.
`This PR is dependent on https://github.com/zephyrproject-rtos/zephyr/pull/53411`

Reply:
I have just deleted it and force-pushed the relevant commit.